tests/cases/unittests/jsDocParsing.ts

```diff
@@ -998,7 +1001,10 @@ module ts {
                         ? JSON.parse(Utils.sourceFileToJSON(v))
                         : v;
                 }, 4);
-                
+
+                // Remove the parserContextFlags from the comparison
+                result = result.replace(/\,\n\s+\"parserContextFlags\": \"JavaScriptFile\"/g, "");

```

Comment:
Can we change `Utils.sourceFileToJSON` in `harness` to exclude that bit (~ line 279)?